### PR TITLE
Translate website into Dutch (nl-NL)

### PR DESCRIPTION
**Pull request by Joery**
I'm pretty sure most of the Dutch population below 50 speaks or at least understands English in some degree but oh well.

If you need any help with translating the actual app into Dutch, hmu on the Paperback Discord.

This also fixes a few spelling errors in the English version.

### CODE REVIEW

**Comment by Joery** [comment]
Everything's translated & Soitora meta fixes included in the Dutch part as well

File: src/nl/help/guides/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+lang: nl-NL
+---
+
+# Aan de slag
+
+## Installatie
+**Paperback** is momenteel _niet_ op **TestFlight** en _vereist_ het gebruik van **AltStore** om op je iPhone of iPad te installeren. In de toekomst zal TestFlight worden gebruikt wanneer de hoofdontwikkelaar een [Apple Developer Account] heeft (https://developer.apple.com/programs/).
+
+::: tip Video tutorial
+Als je liever een video-tutorial gebruikt, heeft Artesians er een mooie gemaakt. Je vindt deze [hier] (https://www.youtube.com/watch?v=n1KRwsxNiWY).
```
Comment (FramboisePi) [inline src/nl/help/guides/getting-started.md:12]: There is an issue with some links. You should remove the space characters for them to work:
 * `[Apple Developer Account](https://developer.apple.com/programs/)`
 * `[hier](https://www.youtube.com/watch?v=n1KRwsxNiWY)`
 * `[GitHub releases-pagina] (https://github.com/FaizanDurrani/Paperback/releases/latest)`
 * `[Afbeelding](https://imgur.com/a/onrwNC8)`

Comment (Joery) [reply src/nl/help/guides/getting-started.md]: Fixed

Comment (FramboisePi) [reply src/nl/help/guides/getting-started.md]: Perfect.

**Pull request merged**